sources/packages/web/src/views/institution/Login.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card elevation="2" class="mx-auto mt-12" max-width="620px" outlined>
+  <v-card elevation="2" class="mx-auto mt-12" max-width="95vh" outlined>
```

Comment:
I believe we must use vw here as it is width. Correct me if I am wrong.

Also, is 95% of width matches figma?